apps/backoffice-v2/src/domains/storage/fetchers.ts

```diff
@@ -39,7 +37,7 @@ export const fetchFileInfoById = async (fileId: string) => {
 };
 export const fetchFileById = async (fileId: string) => {
   const fileInfo = await fetchFileInfoById(fileId);
-  if (fileInfo.fileNameInBucket) {
+  if (fileInfo.fileNameInBucket && import.meta.env.VITE_FETCH_SIGNED_URL === 'TRUE') {
```

Comment:
Is this intentional? (all caps)

Reply:
by default it's true i think lowercased. it should be downcased

Reply:
yes it is uppercased by default